core/src/main/java/org/elasticsearch/action/get/MultiGetRequest.java

```diff
@@ -321,12 +321,32 @@ public MultiGetRequest add(@Nullable String defaultIndex, @Nullable String defau
         String currentFieldName = null;
         while ((token = parser.nextToken()) != XContentParser.Token.END_OBJECT) {
             if (token == XContentParser.Token.FIELD_NAME) {
-                currentFieldName = parser.currentName();
+<<<<<<< HEAD
+		if ("docs".equals(parser.currentName()){ // If statement checks if the field entered matches the word "docs"
+		currentFieldName = parser.currentName();// If it does match, it gets saved to currentFieldName and continues	
+		} else if ("doc".equals(parser.currentName()){
+			throw new IllegalArgumentException("Did you mean'docs'?");
+//If it does not match, the program is stopped with an error message that indicates what needs to be changed		
+=======
+		if ("docs".equals(parser.currentName()){
+		currentFieldName = parser.currentName();		
+		} else if ("doc".equals(parser.currentName()){
+			throw new IllegalArgumentException("Did you mean'docs'?");		
```

Comment:
The check should be that anything except the expected `docs` and `ids` shall throw a general but more descriptive (than before) exception.

Could you also add a test for the new error msg?